hopsworks-web/yo/app/scripts/controllers/newFeaturegroupCtrl.js

```diff
@@ -581,8 +574,7 @@ angular.module('hopsWorksApp')
                 var numberOfPrimary = 0
                 for (i = 0; i < self.onDemandFeaturegroupFeatures.length; i++) {
                     featureNames.push(self.onDemandFeaturegroupFeatures[i].name)
-                    if (self.onDemandFeaturegroupFeatures[i].name === "" || self.onDemandFeaturegroupFeatures[i].name.search(self.hiveRegexp) == -1 ||
-                        self.onDemandFeaturegroupFeatures[i].name.length > self.onDemandFeaturegroupFeatureNameMaxLength) {
+                    if (self.onDemandFeaturegroupFeatures[i].name === "" || self.onDemandFeaturegroupFeatures[i].name.search(self.hiveRegexp) == -1) {
```

Comment:
I think you can get rid of the first part of the if - the regex itself requires at least 1 char.

Reply:
thanks, removed it